lts.json

```diff
@@ -21,21 +21,21 @@
   "ember-cli-sri": "^2.1.0",
   "ember-cli-test-loader": "^1.1.0",
   "ember-cli-uglify": "^1.2.0",
-  "ember-computed-decorators": "~0.2.0",
-  "ember-concurrency": "~0.7.0",
+  "ember-computed-decorators": "~0.2.2",
+  "ember-concurrency": "~0.7.15",
   "ember-css-modules": "~0.5.0",
```

Comment:
Not that - if you *do* install it you can still use SASS exactly as today with a few additions to your build process, correct?  If so, we can keep this